src/ResourceManager/IotHub/Commands.IotHub/Commands.IotHub.csproj

```diff
@@ -170,11 +164,19 @@
       <Link>AzureRM.IotHub.psd1</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="help\Update-AzureRmIotHub.md" />
     <None Include="MSSharedLibKey.snk" />
+    <None Include="packages.config" />
     <None Include="StartupScripts\*.ps1">
       <!-- <CopyToOutputDirectory>Always</CopyToOutputDirectory> -->
     </None>
   </ItemGroup>
+  <ItemGroup>
```

Comment:
You will need to make this change in the netcore csproj as well

Reply:
Done